content/learn/who-uses-amp.yaml

```diff
@@ -3,6 +3,7 @@ $order: 1
 class: about-who
 $view: /views/about-who.html
 $parent: /content/learn/overview.yaml
+$path: /learn/who-uses-amp/
```

Comment:
Grow uses collections to figure out paths. Since the collection's name (`learn/who`) does not match the path (`learn/who-uses-amp`), it is giving incorrect output. So the only options available are to either explicitly state the path as this PR does or change the name of the collection's folder from `who` to `who-uses-amp`. If you decide on the latter option, which might be preferable, it will break a few other places that the old collection path is referenced (`content/includes/menu.yaml`, `views/about-who.html`). Let me know what you decide, I'd be happy to rebase.